qa/python_models/dlpack_test/model.py

```diff
@@ -100,27 +119,56 @@ def test_dlpack_gpu_tensors(self):
             # the same
             pytorch_tensor_dlpack = from_dlpack(pb_tensor.to_dlpack())
             self.assertTrue(torch.all(pytorch_tensor_dlpack == pytorch_tensor))
-
-            # DLPack does not properly support bool type:
-            # https://github.com/google/jax/issues/4719
-            if pytorch_dtype != torch.bool:
-                self.assertTrue(
-                    pytorch_tensor.type() == pytorch_tensor_dlpack.type())
-            else:
-                self.assertFalse(
-                    pytorch_tensor.type() == pytorch_tensor_dlpack.type())
+            self.assertTrue(
+                pytorch_tensor.type() == pytorch_tensor_dlpack.type())
+            
+            # Now we make sure that updated DLPack implementation works
+            # with GPU as well
+            pb_tensor = pb_utils.Tensor.from_dlpack('test_tensor',
+                                                    pytorch_tensor)
+            pytorch_tensor_dlpack = from_dlpack(pb_tensor)
+            self.assertTrue(torch.all(pytorch_tensor_dlpack == pytorch_tensor))
```

Comment:
Same comment as above. `self.assertTrue(torch.equal(pytorch_tensor_dlpack, pytorch_tensor))` I know all of these coding styles were present before your ticket (e.g. the one on line 121), but cleaning them up here for going forward would be good.